domain/model/service/service_test.go

```diff
@@ -26,12 +28,14 @@ type dummyStateCloud struct {
 type dummyState struct {
 	clouds map[string]dummyStateCloud
 	models map[model.UUID]model.ModelCreationArgs
+	users  map[user.UUID]any
```

Comment:
This would be less confusing as a set.Strings. I kept looking to see what was added to the map and all we seem to do it set a a user uuid key value to nil and later check for existence. That's what we typically use set.Strings for in other places.

Reply:
100% correct just forgot it existed.